### PR TITLE
react-redux XHR guideline

### PR DESCRIPTION
**Pull request by phillipemoreira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,7 @@ Code in this package comes from those sources:
 
 A HOC to inject reducers into the store. 
 Based on https://github.com/react-boilerplate/react-boilerplate/blob/master/internals/templates/utils/injectReducer.js
```
Comment (pamepeixinho) [inline README.md]: [total cosmetic stuff] consider add a shortname to this url like `[injectReducer](https://github.com/react-boilerplate/react-boilerplate/blob/master/internals/templates/utils/injectReducer.js)`

File: guidelines.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+2 - A XHR Request is made in order to retrieve a json payload with the user data.
+
+4 - The page changes, not it shows a loading animation.
+
+5 - The requet gets resolved successfully and your json gets parsed into the store.
+
+6 - The animation disappear and finally your user data is drawn instead.
+
+This is a perfec example of where you should use redux-pack, if you do it from scratch without the aid of any mechanism, steps 2 and 4 would have to be manually dispatched inside the XHR promise flow, Add an extra dispatch if you want to handle errors. Redux-pack lets you do all of this within ONE SINGLE redux action dispatch.
```
Comment (pamepeixinho) [inline guidelines.md]: [typo] **perfec**

Comment (pamepeixinho) [reply guidelines.md]: Also,
> promise flow, **Add**

File: guidelines.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+3 - The page changes, not it shows a loading animation.
+
+4 - The requet gets resolved successfully and your json gets parsed into the store.
+
+5 - The animation disappear and finally your user data is drawn instead.
+
+<img align="middle" height="200" src="images/redux-pack-flow.png">
+
+This is a perfec example of where you should use redux-pack, if you do it from scratch without the aid of any mechanism, steps 2 and 4 would have to be manually dispatched inside the XHR promise flow, Add an extra dispatch if you want to handle errors. Redux-pack lets you do all of this within ONE SINGLE redux action dispatch.
```
Comment (pamepeixinho) [inline guidelines.md]: [typo] **perfec**

Comment (pamepeixinho) [reply guidelines.md]: Also, 

> promise flow, **Add** an

**Comment by pamepeixinho** [comment]
Btw, really good job!!! :1st_place_medal:  (and thanks haha)

File: README.md (hunk 1 above)
Comment (phillipemoreira) [reply README.md]: Commenting on stuff I didn't change.. such a peixinho move

**Pull request merged**